docs/howtos/DataPlane Generation - DPG/07tcgcTypes.mdx

```diff
@@ -1407,7 +1426,7 @@ export type AccessFlags = "internal" | "public";
 ```ts
 export interface SdkEndpointType {
   kind: "endpoint";
-  serverUrl?: string;
+  serverUrl: string;
```

Comment:
Is this a breaking change for any of the emitters?

Reply:
No, this was just a fix in documentation of our type: the `serverUrl` has been required this whole time